pr: chore - Improving routes page (#2097)
strapi/documentation: docusaurus/docs/dev-docs/backend-customization/routes.md

- with [policies](#policies), which are a way to block access to a route,
- and with [middlewares](#middlewares), which are a way to control and change the request flow and the request itself.

> Once a route exists, reaching it executes some code handled by a controller (see [controllers documentation](/dev-docs/backend-customization/controllers)). To view all existing routes and their hierarchal order you can run `yarn strapi routes:list`.

Reviewer: ```suggestion
Once a route exists, reaching it executes some code handled by a controller (see [controllers documentation](/dev-docs/backend-customization/controllers)). To view all existing routes and their hierarchal order, you can run `yarn strapi routes:list` (see [CLI reference](/dev-docs/cli)).
```

    {
      method: 'GET',
      path: '/articles/customRoute',

>       handler: 'api::apiName.controllerName.functionName',

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin
```

>       handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-scoped route
```

>       handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-scoped route

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-specific controller
```

    {
      method: 'GET',
      path: '/articles/customRoute',

>       handler: 'api::apiName.controllerName.functionName',

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin
```

>       handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-scoped route
```

>       handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-scoped route

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-specific controller
```

    {
      method: 'GET',
      path: '/articles/customRoute',

>       handler: 'api::apiName.controllerName.functionName',

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin
```

>       handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-specific controller
```

    {
      method: 'GET',
      path: '/articles/customRoute',

>       handler: 'api::apiName.controllerName.functionName',

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-scoped route
```

>       handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-scoped route

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-specific controller
```

    {
      method: 'GET',
      path: '/articles/customRoute',

>       handler: 'api::apiName.controllerName.functionName',

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-specific controller
```

    {
      method: 'GET',
      path: '/articles/customRoute',

>       handler: 'api::apiName.controllerName.functionName',

Reviewer: ```suggestion
      handler: 'api::api-name.controllerName.functionName', // or 'plugin::plugin-name.controllerName.functionName' for a plugin-specific controller
```
